Dependencies/xr/Include/XR.h

```diff
@@ -146,6 +146,24 @@ namespace xr
                     bool PoseTracked{ false };
                 };
 
+                struct Button
+                {
+                    bool Pressed{ false };
+                    bool Touched{ false };
+                    double Value{0};
+                };
+
+                struct GamePad
+                {
+                    using Identifier = size_t;
+                    const Identifier ID{ NEXT_ID++ };
+                    std::vector<float> axes;
```

Comment:
>axes [](start = 39, length = 4)

nit: Axes and Buttons should be capitalized based on other struct definitions preference in capitalizing public fields